src/stan/analyze/mcmc/compute_effective_sample_size.hpp

```diff
@@ -1,7 +1,7 @@
 #ifndef STAN_ANALYZE_MCMC_COMPUTE_EFFECTIVE_SAMPLE_SIZE_HPP
 #define STAN_ANALYZE_MCMC_COMPUTE_EFFECTIVE_SAMPLE_SIZE_HPP
 
-#include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <stan/math/prim/fun/Eigen.hpp>
```

Comment:
This is fine, but below you replace it with `prim.hpp`.

Reply:
Yes, those should just include this file.